pyrk/driver.py

```diff
@@ -215,12 +235,26 @@ def main(args, curr_dir):
     logger.set_up_pyrklog(args.logfile)
     infile = load_infile(args.infile)
     out_db = database.Database(filepath=args.outfile)
+    if not hasattr(infile, 'n_ref'):
+        n_ref = 0
+        ref_lambda = 0
+        ref_rho = 0
+        Lambda_ref = 0
+    else:
+        n_ref = infile.n_ref
+        ref_lambda = infile.ref_lambda
+        ref_rho = infile.ref_rho
+        Lambda_ref = infile.Lambda_ref
     si = sim_info.SimInfo(timer=infile.ti,
                           components=infile.components,
                           iso=infile.fission_iso,
                           e=infile.spectrum,
                           n_precursors=infile.n_pg,
                           n_decay=infile.n_dg,
+                          n_reflector=n_ref,
+                          Lambda_ref=Lambda_ref,
+                          ref_lambda=ref_lambda,
```

Comment:
"ref_lambda" is a list, correct? Thus, it should be "ref_lambdas" or "lambdas_ref" . Use plurality to help users understand the difference between it and similarly named items (e.g. Lambda_ref). Also, in this context, we should think about avoiding "ref" where it doesn't always represent a reflector for all reactors. The model is used for this in our case, but a more formal treatment of the model will refer to this without constraining it to the component name "reflector".